itou/templates/apply/includes/geiq/geiq_diagnosis_details.html

```diff
@@ -70,34 +78,41 @@ <h3>Éligibilité public prioritaire GEIQ non confirmée</h3>
 {% else %}
     {# Diagnosis either does not exist or has expired : this part is for GEIQ only #}
     <hr class="my-4">
-    <div class="row align-items-center">
+    <div class="row">
         <div class="col-12 col-md">
-            <h3 class="mb-2 mb-md-0">Éligibilité public prioritaire GEIQ non confirmée</h3>
+            <h3 class="mb-2 mb-md-0">Éligibilité public prioritaire GEIQ</h3>
         </div>
-        {% if request.user.is_employer %}
-            <div class="col-12 col-md-auto">
+        <div class="col-12 col-sm-auto">
+            {% if diagnosis.allowance_amount %}
+                {% geiq_eligibility_badge is_eligible=True extra_classes="badge-sm" %}
```

Comment:
Dans le bloc `{% else %}` de la ligne 78, correspondant au `{% if diagnosis.is_valid or job_application.state.is_accepted and diagnosis %}` de la ligne 3, on a a priori soit des diag expirés soit absents.

Si en plus `diagnosis.allowance_amount` vaut True, on est donc forcément sur un diag expiré ici.
Je ne pense pas qu'on souhaite afficher `Éligibilité GEIQ confirmée` dans ce cas ?